Tensile/Common.py

```diff
@@ -250,6 +250,7 @@ class DeveloperWarning(Warning):
                                     (11,5,1),
                                     (12,0,0), (12,0,1)] # assembly kernels writer supports these architectures
 
+globalParameters["KeepBuildTmp"] = True                           # remove build artifacts during the build process and build_tmp after build completes
```

Comment:
Maybe better to update the comment.
What you described is the behavior of False case.
By the way, looks like we have similar global parameter option just below this (CleanupBuildFiles).
Any difference?

Reply:
Sure. As for the other option. I would like to remove it and the corresponding code. It has a similar effect but seems unsafe.